libvast/src/format/reader_factory.cpp

```diff
@@ -0,0 +1,71 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2016 The VAST Contributors
```

Comment:
This is a new file, so the copyright should say 2021.

This applies to all the other files in this PR as well.